packages/gcf-utils/test/gcf-bootstrapper.ts

```diff
@@ -1837,8 +1826,11 @@ describe('GCFBootstrapper', () => {
           },
         },
       });
-      runScope.done();
-      sinon.assert.calledOnce(adcStub);
+      // https://github.com/DefinitelyTyped/DefinitelyTyped/issues/36436
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      sinon.assert.calledOnceWithExactly(getServiceStub as any, {
```

Comment:
I never experienced that I need to use `any` for this type of asserts.

Reply:
There's a bug with sinon types and multiple type definitions